src/parseMessage.h

```diff
@@ -0,0 +1,44 @@
+#ifndef PARSE_MESSAGE_H
+#define PARSE_MESSAGE_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+typedef struct MessageParser {
```

Comment:
nit: This isn't really the parser, could call it `MessageBytes` or `RawMessage`, `MessageStream`, or ...

Reply:
I'll give it some thought.  It seems more that "Message" is the wrong word than "Parser", no?

Reply:
Maybe both, that structure's name contains a verb though it doesn't perform any action.  Just a nit, take it or leave it :-)

Reply:
went with Parser